### ISSUE TITLE
Example application using lager

### ISSUE THREAD

**Issue by pr0g** [arximboldi/lager#110]
Hi @arximboldi,

TLDR - I used your library and thought you might find this interesting - [lager-hierarchy](https://github.com/pr0g/lager-hierarchy).

This isn't an issue so please feel free to close after you've read this.

I've been playing around with an incredibly simple library that represents an entity hierarchy. The base library is available here - [hierarchy](https://github.com/pr0g/hierarchy). It includes a simple macOS and Windows console application using the library. I made another example that uses Qt here - [qt-hierarchy](https://github.com/pr0g/qt-hierarchy) and have an ImGui version as part of this app ([relevant file](https://github.com/pr0g/sdl-bgfx-imgui-as_1d-nonlinear-transformations/blob/main/hierarchy-imgui.cpp)).

I saw the awesome presentation you gave at C++ on Sea 2020 about `lager` and have finally got around to giving it try. This mini example again consumes the hierarchy library but uses `lager` to drive the UI - [lager-hierarchy](https://github.com/pr0g/lager-hierarchy).

It's incredibly simple but I thought it might be interesting to share. There's still a lot more I'd like to add to it (one thing might be swapping out my use of `std` containers with `immer` in future).

I'd love to hear any feedback, suggestions or advice!

Thanks for putting together such a cool and interesting library and for the excellent documentation and guide!

Cheers,

Tom

**Comment by arximboldi** [comment]
Hi! Thanks for your interest in the library. I think your example looks nice. Only issue is, as you pointed out, that you seem to be using `std` containers in the model, which are actually quite inefficient to use within Lager models.
